hazelcast/src/test/java/com/hazelcast/scheduledexecutor/ScheduledExecutorServiceSlowTest.java

```diff
@@ -370,4 +373,27 @@ public void cancelUninterruptedTask_waitUntilRunCompleted_killMember_checkStatus
         assertTrue(future.isDone());
     }
 
+    @Test
+    public void reschedulingAfterMigration_whenCurrentNodePreviouslyOwnedTask() {
+        TestHazelcastInstanceFactory factory = createHazelcastInstanceFactory();
+        HazelcastInstance first = factory.newHazelcastInstance();
+
+        int tasksCount = 1000;
+        final IScheduledExecutorService scheduler = getScheduledExecutor(new HazelcastInstance[] {first }, "scheduler");
```

Comment:
I don't want to remove the whole method. But this test looks like it doesn't need it, since it requires to construct an array (which looks more complex than just retrieving the scheduler).